src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -156,6 +156,11 @@ bool MulticopterLandDetector::_get_ground_contact_state()
 		return true;
 	}
 
+	// if it was already landed and didn't get out of low thrust, it has to be still on the ground
```

Comment:
Why is this not in the `_get_landed_state`? When it's in `ground_contact` the state can still fall back and forth and there's no guarantee that the thrust stays low in ground contact. In fact that's something I changed in #11172

Reply:
You're right, I'll move it to `_get_landed_state`. That way, only freefall can change the state.